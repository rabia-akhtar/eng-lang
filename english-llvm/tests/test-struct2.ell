struct Doc {
	string File;
	int Word_count;
};

struct Doc returnDoc(){
  struct Doc doc;
  doc.File = "this is an essay";
  doc.Word_count = 20;
  print_string(doc.File);
  print(doc.Word_count);
  return doc;
}

int main()
{
  struct Doc essay = returnDoc();
  print_string(essay.File);
  print(essay.Word_count);
  return 0;
}