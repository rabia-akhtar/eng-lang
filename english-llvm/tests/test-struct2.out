this is an essay
20
this is an essay
20